src/addresses_ns2.sol

```diff
@@ -0,0 +1,22 @@
+contract Addresses {
+	address constant public ROOT = 0x53b2d22d07E069a3b132BfeaaD275b10273d381E;
+	address constant public TINLAKE_CURRENCY = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+	address constant public TITLE = 0x9Ab3ada106afdFAE83f13428e40da70b3A22C50C;
+	address constant public PILE = 0x3eC5c16E7f2C6A80E31997C68D8Fa6ACe089807f;
+	address constant public SHELF = 0x7d057A056939bb96D682336683C10EC89b78D7CE;
+	address constant public COLLECTOR = 0x62f290512c690a817f47D2a4a544A5d48D1408BE;
+	address constant public FEED = 0x41fAD1Eb242De19dA0206B0468763333BB6C2B3D;
+	address constant public JUNIOR_OPERATOR = 0x4c4Cc6a0573db5823ECAA1d1d65EB64E5E0E5F01;
+	address constant public SENIOR_OPERATOR = 0x230f2E19D6c2Dc0c441c2150D4dD9d67B563A60C;
+	address constant public JUNIOR_TOKEN = 0x961e1d4c9A7C0C3e05F17285f5FA34A66b62dBb1;
+	address constant public SENIOR_TOKEN = 0xE4C72b4dE5b0F9ACcEA880Ad0b1F944F85A9dAA0;
+	address constant public JUNIOR_MEMBERLIST = 0x42C2483EEE8c1Fe46C398Ac296C59674F9eb88CD;
+	address constant public SENIOR_MEMBERLIST = 0x5B5CFD6E45F1407ABCb4BFD9947aBea1EA6649dA;
+	address constant public COORDINATOR = 0xcC7AFB5DeED34CF67E72d4C53B142F44c9268ab9;
```

Comment:
This one is old, it was swapped again in July 2021.

```suggestion
	address constant public COORDINATOR = 0x22a1caca2EE82e9cE7Ef900FD961891b66deB7cA;
```

Reply:
good catch